src/Components/MyLists/MyLists.scss

```diff
@@ -0,0 +1,45 @@
+@use '../../styles/_variables' as v;
+@use '../../styles/_mixins' as m;
+
+
+.list-dropdown-container {
+  @include m.flexbox(column, wrap, center, center, center);
+
+  .instructions {
+    font-size: 1.5em;
+    color: v.$teal;
+    margin: 2% 5%;
+    text-align: center;
+  }
+
+  .list-dropdown {
+    border: 1px solid v.$palePeach;
+    border-radius: 0.2em;
+    color: v.$orange;
+    background-color: v.$transparentPeach;
+    max-width: 10em;
+    min-height: 2em;
+    font-size: 1.5em;
+
+    &:hover {
+      background-color: v.$orange;
+      color: v.$offWhite;
+    }
+  }
+}
+
+
+
+//desktop requirements
+@media screen and (min-width: 40em) {
+  .hidden-desktop {
+    display: none;
+  }
+}
+
+//mobile requirements
+@media screen and (max-width: 40em) {
```

Comment:
I love the change you made to your menu for mobile! Just make sure its still as easy to get to as it is on desktop (right now a lot of scrolling has to happen first).